### DISCUSSION TITLE
Confusion over Licensing

### DISCUSSION THREAD

**Discussion by hyandell** [yandeu/five-server#93] [Help / Questions]
Hi Yannick,

I was reading the license for your project and I'm very confused about who can use it. The license file in 2c) says "You are NOT permitted to ... use ... the Software for any commercial or non-commercial purpose". Doesn't that mean nobody can use the software?

I also don't see the original MIT license being referred to - did you accidentally lose that attribution to the original content?

**Comment by yandeu** [comment]
I admit ot sounds strange. But it is more refering to "Reproduce, copy, distribute, resell" than to use. I'm not a lawyer, but it meant to be free to use.

Five-server is not really a fork, but a rewrite. Some files in the source still refer to the MIT license by tapio (or others).

**Reply by hyandell** [reply]
Thanks Yannick.

I'm also not a lawyer; so I only have questions and rough understanding rather than any advice :)

Typically I'd understand 'copy' to include the permission to download or move from computer to computer once downloaded. So for the computer concept of 'use', the 'copy' term usually seems essential. It does feel like your current license text does not match your intentions.

**Reply by yandeu** [reply]
Yes, but I couldn't find a matching license so I chose this one. I thought no one (who just want to use the software from npm or vscode) would read it. But I agree, I probably should adjust it.
